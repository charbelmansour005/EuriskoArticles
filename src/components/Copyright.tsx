import {View, Text} from 'react-native';
import {themeColors} from '../helpers/themeColors';
import React from 'react';

type Props = {};

const Copyright = (props: Props): JSX.Element => {
  return (
    <Text style={{fontSize: 12, color: themeColors.grey, textAlign: 'center'}}>
      {'Copyright Â© '} Eurisko Mobility {new Date().getFullYear()}
    </Text>
  );
};

export default Copyright;
