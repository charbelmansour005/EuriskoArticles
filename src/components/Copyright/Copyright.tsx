import {Text} from 'react-native'
import {themeColors} from '../../helpers/themeColors'
import React from 'react'

const Copyright = (): JSX.Element => {
  return (
    <Text
      style={{fontSize: 12, color: themeColors.grey, textAlign: 'center'}}
      testID="copyright">
      {'Copyright Â© '} Eurisko Mobility {new Date().getFullYear()}
    </Text>
  )
}

export default Copyright
