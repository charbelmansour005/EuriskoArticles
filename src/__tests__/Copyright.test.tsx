import React from 'react'
import renderer from 'react-test-renderer'
import Copyright from '../components/Copyright/Copyright'

test('renders correctly', async () => {
  const tree = renderer.create(<Copyright />).toJSON()
  expect(tree).toMatchSnapshot()
})
