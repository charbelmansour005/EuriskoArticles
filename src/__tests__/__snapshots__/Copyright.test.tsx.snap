// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<Text
  style={
    Object {
      "color": "#748c94",
      "fontSize": 12,
      "textAlign": "center",
    }
  }
>
  Copyright Â© 
   Eurisko Mobility 
  2022
</Text>
`;
